kuma/javascript/src/search-results-page.jsx

```diff
@@ -111,7 +110,7 @@ function Results({
                         }}
                     />
                     <div className="result-url">
-                        <a href={path}>{url}</a>
+                        <a href={path}>{result.slug}</a>
```

Comment:
```suggestion
                        <a href={path}>{path}</a>
```
Otherwise there might be to identical strings there. E.g. `Web/HTML/Css` and `Web/HTML/Css`. But if you hover over the links you see that the actual URLs are `/en-US/docs/Web/HTML/Css` and `/sv-SE/docs/Web/HTML/Css`.
The Wiki site-search has slightly better messaging but basically, if you're using https://developer.mozilla.org/sv-SE/search?q=stuff&topic=apps&topic=html&topic=css&topic=js&topic=api&topic=canvas&topic=svg&topic=webgl&topic=mobile&topic=webdev&topic=http&topic=webext&topic=standards it will search Elastisearch for `{q: "stuff", locale: 'sv-SE' || 'en-US'}` so you'll get a little for each locale.

Reply:
Done :) Hopefully breadcrumbs are somewhere in our future, but this is a small improvement in the meantime.